charts/kube-master/templates/reject-proxy-config.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.rejectProxy.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "master.fullname" . }}-reject-proxy
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
I think this can be a `Role` and `RoleBinding`, no? The reject proxy only needs to access a ConfigMap in the same namespace as itself is running afais.

Reply:
I changed this.